Update CircleCI and other settings, add howto (#2)
vacuumlabs/init-handbook: README.md

# Base for handbooks

Steps to run locally:

1. Install ruby [rvm](https://rvm.io/)

> 2. Run `bundle` to install dependencies

Reviewer: ```suggestion
1. Run `bundle` to install dependencies
```

> 3. Run `bundle exec guard` to start watching changes to the asciidoc files

Reviewer: ```suggestion
1. Run `bundle exec guard` to start watching changes to the asciidoc files
```

> 4. Open `dist/index.html` to view your handbook

Reviewer: ```suggestion
1. Open `dist/index.html` to view your handbook
```

> For interactions with the guard shell, see [link](https://github.com/guard/guard/wiki/Interacting-with-Guard).

Reviewer: ```suggestion
> **Note**: For interactions with the guard shell, see [the Interacting with Guard documentation](https://github.com/guard/guard/wiki/Interacting-with-Guard) e.g. `all` will rebuild the project.
```

> (e.g. `all` will rebuild the project)

Reviewer: ```suggestion

```

Author: ?

To just simply build the handbook run:

> ```

Reviewer: ```suggestion

```sh
```
